testing_app/test/home_test.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+import 'package:testing_app/models/favorites.dart';
+import 'package:testing_app/screens/home.dart';
+
+Widget createHomeScreen() => ChangeNotifierProvider<Favorites>(
+      create: (context) => Favorites(),
+      child: MaterialApp(
+        home: HomePage(),
+      ),
+    );
+void main() {
```

Comment:
nit: add new line between `createHomeScreen` and `main`. I feel like this should have been picked up by `flutter format`...

Reply:
I wonder why it didn't. I even tried it again right now. Finally ended up adding it manually. I think ```flutter format``` doesn't add new lines on its own.

Reply:
there is always something to add =)